admin/sections/config/chat.tsx

```diff
@@ -18,11 +18,16 @@ export const ChatConfig = ({
   form: any;
   isSubmitting: boolean;
 }) => {
+  const chatConfigFields = ["system_prompt", "conversation_starters"];
+  const fieldsChanged = chatConfigFields.some(
+    (field) => form.formState.dirtyFields[field],
```

Comment:
When is dirty fields updated? On focus lost? Because that's when we want to do the submit

Reply:
The form dirty fields are updated once the new value is different from the initial value. I think it makes more sense than on focus lost.